src/fragments/lib-v1/start/getting-started/reactnative/setup.mdx

```diff
@@ -0,0 +1,138 @@
+## Create a new project
```

Comment:
Should remove the React Native getting started guide as it's not supported for Dev Preview